source/sfp/4_vegetation/sfp_407_nl_bhm-opnameschaal/12_appendices.Rmd

```diff
@@ -0,0 +1,80 @@
+# (APPENDIX) Bijlagen {.unnumbered}
+
+# Bijlage Visuele ondersteuning beheermonitoringschaal {#bijlage-visueel}
+
+De figuren geven voor elke code uit de beheermonitoringschaal een minimale en maximale visuele indruk van de waargenomen aantallen die men waarneemtbij het doorkruisen van een inventarisatie-eenheid (Figuur \@ref(fig:visueleondersteuning)).
+ZS = zeer schaars, S = schaars, WT = weinig talrijk, T = talrijk, B = bedekkend, KB = kwartbedekkend, HB = halfbedekkend en D = dominant.
+
+(ref:visueleondersteuning) Visuele ondersteuning van de codes van de beheermonitoringschaal. ZS = zeer schaars, S = schaars, WT = weinig talrijk, T = talrijk, B = bedekkend, KB = kwartbedekkend, HB = halfbedekkend en D = dominant.
+
+```{r visueleondersteuning, fig.cap="(ref:visueleondersteuning)", fig.show=ifelse(is_html_output(), "hold", "asis"), fig.align="center"}
+include_graphics(
+  path = file.path(
+    path_to_protocol,
+    "media",
+    c("zeerschaars-1.png", 
+      "schaars-1.png", 
+      "weinigtalrijk-1.png",
+      "talrijk-1.png",
+      "bedekkend-1.png",
+      "kwartbedekkend-1.png",
+      "halfbedekkend-1.png",
+      "dominant-1.png"
+    )
+  )
+)
+```
+
+Figuur \@ref(fig:rand-afwijkend) toont enkele voorbeelden hoe de beheermonitoringschaal omgaat met situaties waarbij een soort alleen voorkomt in lokaal afwijkende vegetatie of randsituaties (meer dan twee klassen verschil in bedekkingschaal tussen de afwijkende vegetatievlek of de randzone en de rest van het perceel of habitatvlek).
+
+(ref:rand-afwijkend) Visualisatie van bedekking van een soort die enkel voorkomt in een zone met lokaal afwijkende vegetatie of in een randzone.
+
+```{r rand-afwijkend, fig.cap="(ref:rand-afwijkend)", fig.align="center"}
+include_graphics(
+  file.path(
+    path_to_protocol,
+    "media",
+    c(
+      "lokaalrand-1.png"
+    )
+  )
+)
+```
+
+# Bijlage Indicatieve omzetting naar een bedekkingspercentage {#bijlage-omzetting}
+
+(ref:indcover) Indicatieve omzetting van bedekkingspercentages naar absolute bedekkingen
+
+```{r indcover, fig.cap="(ref:indcover)", fig.align="center"}
+include_graphics(file.path(
+  path_to_protocol, 
+  "media",
+  c("indcover_bhm.png")))
+```
+
+Voor de lokale varianten van een categorie (X/2, X/4, X/8 en XR) wordt de standaard procentuele bedekking van de categorie vermenigvuldigd met een respectievelijke factor 3/8, 3/16, 1/16 en 1/16.
+Bv.
+WT=0.030%, WT2=0.01125%, WT4=0.005625%, WT8=0.001875% en WTR=0.001875%.
+
+# Bijlage Veldformulier {#bijlage-veldformulier}
+
+(ref:veldform) Voorbeeld van een veldformulier waarbij de beheermonitoringschaal is gebruikt
+
+```{r veldform, fig.cap="(ref:veldform)", fig.align="center"}
+include_graphics(file.path(
+  path_to_protocol, 
+  "media",
+  c("veldform.png")))
+
+```
+
+# Bijlage Projectfiche en defaultwaarden {#bijlage-projectfiche}
+
+```{r projectfiche, fig.cap="(ref:projectfiche)", fig.align= "center"}
```

Comment:
(ref:projectfiche) is niet gedefinieerd. Je moet nog een aparte paragraaf toevoegen die je begint met (ref:projectfiche) gevolgd door de hoofdingtekst.

Reply:
??? wat moet hier nog gebeuren??? bijlage is eruit

Reply:
Niets, als de bijlage weg is, is het probleem opgelost.